recipe/config.mk.patch

```diff
@@ -0,0 +1,20 @@
+--- xgboost-0.7/xgboost/make/config.mk.orig
++++ xgboost-0.7/xgboost/make/config.mk
+@@ -17,12 +17,14 @@
+ #----------------------------------------------------
+
+ # choice of compiler, by default use system preference.
+-# export CC = gcc
+-# export CXX = g++
++export CC = clang
++export CXX = clang++
+ # export MPICXX = mpicxx
+
++USE_OPENMP = 0
```

Comment:
Why is OpenMP always off?

Reply:
Uh. Yes, this is wrong; a leftover from a local build. I'll fix it up.